dds_web/security/auth.py

```diff
@@ -49,7 +49,8 @@ def get_user_roles(user):
 
 
 def get_user_roles_common(user):
-    return user.role
+    flask.current_app.logger.debug(f"ROLE: {getattr(user, 'role')}")
+    return getattr(user, "role")
```

Comment:
Should user.role ever be missing? I guess that's why you have the `getattr` instead of what was before...?

Reply:
I had a bug, that's why this was changed. Will change back. `role` should always be set.